src/components/Account/AccountModule.js

```diff
@@ -28,20 +28,13 @@ function AccountModule() {
   const toggle = useCallback(() => setOpened((opened) => !opened), [])
 
   useEffect(() => {
-    let timer
```

Comment:
Just curious, why are we deleting this?

Reply:
This was added as workaround for an animation issue [here](https://github.com/aragon/court-dashboard/pull/375/files#r463716108) and setting this value to `null` after the timeout is what makes the module unable to determine the type of wallet that is connecting. I gave it a good amount of testing and saw no animation issues without it.

Reply:
awesome! :)

Reply:
But keep an eye out, if all is well we can suggest the same changes in our other repos :)